ajax/debug.php

```diff
@@ -67,5 +67,47 @@
     die();
 }
 
+if (isset($_GET['action'])) {
+    $action = $_GET['action'];
+    if ($action === 'get_itemtypes') {
+        $loaded = get_declared_classes();
+        $glpi_classes = array_filter($loaded, static function ($class) {
+            return is_subclass_of($class, 'CommonDBTM');
```

Comment:
Abstract classes should be removed from the list.

```suggestion
            if (!is_subclass_of($class, 'CommonDBTM')) {
                return false;
            }

            $reflection_class = new ReflectionClass($class);
            if ($reflection_class->isAbstract()) {
                return false;
            }

            return true;
```